CI/remoteLayerCaculation.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+cd docs/private_tangle
+
+ROOT_TREE_LOCAL=`cat data/layers/layer.0.csv`
+echo "starting Signing server"
+./11_run_signature_source_server.sh &
+sleep 10
+
+echo "starting remote layer calculation"
+./21_calculate_layers_remote.sh
+
+echo "compare results"
+ROOT_TREE_REMOTE=`cat data/layers/layer.0.csv`
+diff ../data/layers/layer.0.csv data/layers/layer.0.csv
```

Comment:
I see ROOT_TREE_REMOTE is the exact same command as ROOT_TREE_LOCAL
A line down you are comparing files in different location.

I think I am not following the test...

Reply:
I just saw this in the log:
`diff: ../data/layers/layer.0.csv: No such file or directory`

Reply:
this line was meant to be removed. removing now.
(see the test just after - ROOT_TREE_REMOTE vs. ROOT_TREE_LOCAL)
```suggestion
```